### PR TITLE
Preserve Music and genre data

### PR DESCRIPTION
**Pull request by momotsow**
In this milestone, I have created the following changes:

- Display all created music options
- Display all created genres
- Add music album
- Create a schema.sql file with tables that will be analogical to the structure of the classes
- Preserve Music data in Json file
- Preserve Genre data in Json file

### CODE REVIEW

**Review by yosefHesham — CHANGES_REQUESTED** [review]
Hey @momotsow  !
You have done a great job !, however there is some changes.

Please review the below comments

File: database/schema.sql [hunk 1]
```diff
@@ -20,6 +33,20 @@ CREATE TABLE author(
 	last_name varchar(150),
 );
 
+<<<<<<< HEAD
```
Comment (yosefHesham) [inline database/schema.sql:36]: Remove this <<<HEAD please

File: database/schema.sql [hunk 2]
```diff
@@ -20,6 +33,20 @@ CREATE TABLE author(
...
+);
+
+CREATE TABLE Music_Album(
+  id SERIAL PRIMARY KEY,
+  publish_date DATE,
+  name VARCHAR(100),
+  on_spotify BOOLEAN,
+  item_id INT,
+  CONSTRAINT item_fk FOREIGN KEY (item_id) REFERENCES Items (id)
+=======
```
Comment (yosefHesham) [inline database/schema.sql:49]: Also this

File: database/schema.sql [hunk 3]
```diff
@@ -32,4 +59,5 @@ CREATE TABLE book(
   FOREIGN KEY (label_ID) REFERENCES label(ID),
   FOREIGN KEY (author_ID) REFERENCES author(ID),
   FOREIGN KEY(genre_ID) REFERENCES genre(ID)
+>>>>>>> eb22764f69c9f5096ca875f6940f701b86c68e22
```
Comment (yosefHesham) [inline database/schema.sql:62]: the last thing =D

**Review by vanelnw — APPROVED** [review]
# STATUS: MILESTONE APPROVED 👍 

Hi @momotsow   , 
### Great job !!!

Your task is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  No linter errors
:heavy_check_mark:  Very descriptive and detailed PR

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**